src/components/landing-hero/components/hero/index.tsx

```diff
@@ -63,15 +64,16 @@ function CertificationsHeroText({
 				className={s.textHeading}
 				dangerouslySetInnerHTML={{ __html: heading }}
 			/>
-			<p
-				className={s.textDescription}
-				dangerouslySetInnerHTML={{
-					__html: mitigateWidows(description, 18),
-				}}
-			/>
+			{description?.length ? (
+				<p
+					className={s.textDescription}
+					dangerouslySetInnerHTML={{
+						__html: mitigateWidows(description, 18),
+					}}
+				/>
+			) : null}
```

Comment:
There isn't a description for the HVD landing page yet, so I've made this optional.